### ISSUE TITLE
Lipid laden macrophages

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5560]
Preferred term label:
Lipid laden macrophages

Synonyms
Fat stain positive macrophages
Foamy macrophages

Definition (free text, please give PubMed ID)
Accumulation of lipids in macrophages with droplet-shaped fat inclusions
Costabel U. Atlas of bronchoalveolar lavage. London: Chapman & Hall; 1998. ISBN0412792702.

Parent term (use hpo.jax.org/app)
Abnormalities of alveolar macrophages

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

Your nano-attribution (ORCID)
@GrieseAG
@katrinknoflach

**Comment by pnrobinson** [comment]
Label Lipid-laden macrophages in BAL

**Comment by pnrobinson** [comment]
added as HP:0032978
